### PR TITLE
Adding non admin passed cases for network stress

### PR DESCRIPTION
**Pull request by mbpavan**
Adding non admin passed cases for network stress

### CODE REVIEW

**Comment by e-minguez** [comment]
Please add the relevant notes in the README file (https://github.com/e-minguez/ose-tests/blob/master/README.md#tests & https://github.com/e-minguez/ose-tests/blob/master/README.md#run-the-tests) as well as adding the new ones to the all.txt file.

**Comment by mbpavan** [comment]
Added the changes. all.txt remains same even with network stress stuff. Also please check if we need any changes to the image for network stress.

File: README.md [hunk 1]
```diff
@@ -124,7 +124,8 @@ The container image includes a few lists proven to be successfully executed with
 
 * [openshift-conformance-minimal](tests-lists/openshift-conformance-minimal.txt). Based on the openshift/conformance/parallel/minimal tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance/parallel/minimal" | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
 * [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```
Comment (e-minguez) [inline README.md]: Can you modify this as well?

File: tests-lists/openshift_network_stress.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"[sig-network] DNS should support configurable pod DNS nameservers [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
```
Comment (e-minguez) [inline tests-lists/openshift_network_stress.txt]: Please rename the file to `openshift-network-stress.txt`

File: README.md [hunk 3]
```diff
@@ -124,7 +124,8 @@ The container image includes a few lists proven to be successfully executed with
...
-* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [openshift-network-stress](tests-lists/openshift_network_stress.txt).  Based on the openshift/network/stress tests suite (`openshift-tests run  openshift/network/stress | grep  -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```
Comment (e-minguez) [inline README.md]: Rename the file to `openshift-network-stress`

**Review by e-minguez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mbpavan) [reply README.md]: Done! We cannot modify for openshift/conformance/minimal as it is not a part of the main test suit. So modified for openshift/conformance only.

File: tests-lists/openshift_network_stress.txt (hunk 2 above)
Comment (mbpavan) [reply tests-lists/openshift_network_stress.txt]: Done

File: README.md (hunk 3 above)
Comment (mbpavan) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -123,8 +123,9 @@ oc adm policy add-cluster-role-to-user self-provisioner-namespaces nonadmin
...
 
 * [openshift-conformance-minimal](tests-lists/openshift-conformance-minimal.txt). Based on the openshift/conformance/parallel/minimal tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance/parallel/minimal" | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
-* [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
-* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
+* [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run openshift/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```
Comment (e-minguez) [inline README.md]: I guess you need to add the `--dry-run` if you only need to extract the list of the tests right?

File: README.md [hunk 5]
```diff
@@ -123,8 +123,9 @@ oc adm policy add-cluster-role-to-user self-provisioner-namespaces nonadmin
...
+* [kubernetes-conformance](tests-lists/kubernetes-conformance.txt).  Based on the kubernetes/conformance tests suite (`openshift-tests run kubernetes/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```
Comment (e-minguez) [inline README.md]: I guess you need to add the `--dry-run` if you only need to extract the list of the tests right?

File: README.md [hunk 6]
```diff
@@ -123,8 +123,9 @@ oc adm policy add-cluster-role-to-user self-provisioner-namespaces nonadmin
...
+* [openshift-network-stress](tests-lists/openshift-network-stress.txt).  Based on the openshift/network/stress tests suite (`openshift-tests run  openshift/network/stress | grep  -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```
Comment (e-minguez) [inline README.md]: I guess you need to add the `--dry-run` if you only need to extract the list of the tests right?
Also, the `grep` command misses the `-v` parameter

**Review by e-minguez — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (mbpavan) [reply README.md]: I Agree. Done.

File: README.md (hunk 5 above)
Comment (mbpavan) [reply README.md]: I Agree. Done.

File: README.md (hunk 6 above)
Comment (mbpavan) [reply README.md]: I Agree. Done. Oops - with the -v param!

**Pull request merged**